assets/lokomotive-kubernetes/bootkube/resources/charts/kube-apiserver/templates/kube-apiserver.yaml

```diff
@@ -107,7 +101,7 @@ spec:
           --service-cluster-ip-range={{ .Values.apiserver.serviceCIDR }} \
           --tls-cert-file=/etc/kubernetes/secrets/apiserver.crt \
           --tls-private-key-file=/etc/kubernetes/secrets/apiserver.key \
-          --token-auth-file=/run/kube-apiserver/token-auth-file \
+          --token-auth-file=/etc/kubernetes/secrets/token-auth-file \
```

Comment:
I don't see where the secret is mounted... Is not mounted here? I see only this: https://github.com/kinvolk/lokomotive/pull/353/commits/dd41c95ae611e8e0cd819b225209af0181545b4d#diff-12be5db389a0fefe0038d10211a890b4R242-R249 and don't see this mounted.

Why is that the new path?

Reply:
The secret is moved from the runtime directory to the secret object, which is always mounted.

From here: https://github.com/kinvolk/lokomotive/pull/353/files#diff-12be5db389a0fefe0038d10211a890b4L250

To here: https://github.com/kinvolk/lokomotive/pull/353/files#diff-0b161dbfa83ace6b4646c7465803f021R29

Reply:
>Why is that the new path?

This is the new path, because this is where rest of the secrets is mounted.

Reply:
Ohh, okay, doing a local checkout of the branch I can find where it is. Thanks!